docs/fleet/provision.md

```diff
@@ -138,7 +140,10 @@ The Viam Agent is installed as a `systemd` service named `viam-agent`.
 With the Viam Agent installed, your machine will either connect to a local WiFi network or will create its own WiFi hotspot, depending on your configuration.
 
 - If you include a `viam-server` configuration file on your machine, located at <file>/etc/viam.json</file>, which includes a WiFi network and password to connect to, the Viam Agent will connect to the network automatically when in range.
-- If you did not include this file, or the configured WiFi network is not available when your machine comes online, the Viam Agent will create its own WiFi hotspot.
+- If you did not include this file, or the configured WiFi network is not available when your machine comes online, the Viam Agent will create its own WiFi hotspot and request WiFi credentials:
```

Comment:
@ale7714 thought this might be more accurate placement than what I had in first commit, lmk your thoughts? Do agent docs need to be updated at all to account for this as the expected flow?

Reply:
I really like it, it has a good flow with the previous info!

No agent docs update needed.

Reply:
Sounds good, thank you! @ale7714